pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>com.coveo</groupId>
     <artifactId>feign-error-decoder</artifactId>
-    <version>1.4.1-SNAPSHOT</version>
+    <version>1.5.0-SNAPSHOT</version>
```

Comment:
why snapshot?

Reply:
Because I'll be preparing a next release with the PR I got from another person